
# Stool Swab Tissue Analysis

## Transparency and Reproducibility

This repository contains:         

 * Counts files produced by processing the sequences, see folder: `data/counts`
 * Metadata files, see folder: `data/metadata`
 * Configuration file to run the analysis component of the project, including all Rcode, see folder: `Analysis`
 * Results of the analysis (figures and tables), see folder: `results`
 
See `Analysis/README` for instructions on how to repeat the entire analysis on your own machine, thus reproducing all figures and tables.  A review module was added to the pipeline to facilitate readers and reviewers comparing the reproduced output the the corresponding components in the manuscript.

## Manuscript

Title:                
_On the robustness of inference of association with the gut microbiota in stool, swab and mucosal tissue samples_

Authors:               
Shan Sun, Anthony A. Fodor

This article is a preprint and has not been peer-reviewed. It reports new medical research that has yet to be evaluated and so should not be used to guide clinical practice.

## Abstract 

The gut microbiota plays an important role in human health and disease. Stool, swab and mucosal tissue samples have been used in individual studies to survey the microbial community but the consequences of using these different sampling methods are not completely understood. We previously reported differences in microbial community composition with 16S rRNA gene amplicon sequencing between stool, swab and mucosal tissue samples. Here, we extend this study to a larger cohort and performed shotgun metagenome sequencing of 1,397 stool, swab and mucosal tissue samples from 240 subjects. Consistent with previous results, taxonomic composition of stool and swab samples was distinct, but still more similar to each other than mucosal samples, which had a radically different community composition, characterized by a high relative abundance of mucus metabolizers Bacteroides and Subdoligranulum, as well as bacteria with higher tolerance for oxidative stress such as Escherichia. As has been previously reported, functional profiles were more uniform across sample types than taxonomic profiles with differences between stool and swab samples smaller, but mucosa samples remaining distinct from the other two types. When used for inference in association with host phenotypes of age, sex, body mass index (BMI), antibiotics and non-steroidal anti-inflammatory drugs (NSAIDs) use, hypothesis testing using either stool or swab gave broadly similar results, but inference performed on mucosal samples gave results that were generally less consistent with either stool or swab. Our publicly available dataset represents an important resource for determination of how inference can change for taxa and pathways depending on the choice of where to sample in the gut.

**Trial Registration** clinicaltrials.gov, number NCT01105169. Registered 16 April 2010, https://www.clinicaltrials.gov/ct2/show/NCT01105169